test/test_images/emptydir/emptydir.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
We need to add this test image into `Makefile` https://github.com/openshift/knative-serving/blob/main/Makefile#L6 and openshift/release repo?

Reply:
Hmm... `Makefile` is fine. Then, why I cannot see the test image in https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_knative-serving/877/pull-ci-openshift-knative-serving-release-v0.24.0-4.6-images/1429737018711085056 :thinking:

Reply:
Chicken/egg problem, I think? We need this on the branch first, then I can setup CI to generate the image, then we can run the tests as you mentioned above.
I'll be adding it to main and release-next then too.

Reply:
Alright, then let's go merge.